resources/views/components/dropdown.blade.php

```diff
@@ -27,7 +27,10 @@
         {{ $trigger }}
     </x-dropdown-trigger>
 
-    <div class="dropdown-menu {{ $dropdownClass ?? '' }}" aria-labelledby="dropdownTrigger{{ $id }}">
+    <div
+        class="dropdown-menu {{ $dropdownClass ?? '' }} z-10 sm:z-auto"
+        aria-labelledby="dropdownTrigger{{ $id }}"
+    >
```

Comment:
I'm still seeing this. Do I need to do more than what's done by `.hooks/post-merge`? Alto tried Ctrl+F5
![image](https://github.com/user-attachments/assets/08de0dfb-a9f8-400a-b786-ea62993e975f)

Reply:
Something was wrong with my local environment. I was seeing a behavior that wasn't actually real due to local modifications in the browser itself that I didn't realize were persisting.

I've made some tweaks in latest that should have things working now.